mbank_parser.py

```diff
@@ -0,0 +1,155 @@
+
+    def convert_row(self, row):
+        new_row = row.split(';')[:-2]
+        # remove Data księgowania
+        new_row.pop(1)
+
+        self.format_expense_amount(new_row)
+        self.merge_accountid_with_memo(new_row)
+        # REGULARNE OSZCZĘDZANIE
+        self.rename_regularne_oszczedzanie(new_row, KONTO_REGULARNE)
+        # Credit Card pay up
+        self.rename_credit_payup(new_row, KARTA)
+        # Kapitalizacja & Odsetki
+        self.replace_other(new_row, KAPITALIZACJA)
+        self.replace_other(new_row, PODATEK_ODSETKI)
+        # if Payee empty, move Memo there
+        self.populate_payee_if_empty(new_row)
+        # transfer to accounts
+        self.rename_internal_transfer(new_row, KONTO_1)
+        self.rename_internal_transfer(new_row, KONTO_2)
+        self.rename_internal_transfer(new_row, KONTO_3)
+
+        return new_row
+
+    def format_expense_amount(self, row):
+        row[self.AMOUNT_COL] = row[self.AMOUNT_COL].replace(',', '.').replace(' ', '')
+
+    def populate_payee_if_empty(self, row):
+        if row[self.PAYEE_COL] == '"  "':
+            row[self.PAYEE_COL] = row[self.MEMO_COL]
+            row[self.MEMO_COL] = ''
+
+    def merge_accountid_with_memo(self, row):
+        if row[self.NUMER_KONTA_COL] != '':
+            row[self.MEMO_COL] = row[self.MEMO_COL] + row[self.NUMER_KONTA_COL]
+
+    def rename_internal_transfer(self, row, account):
+        if INTERNAL_TRANSFER in row[self.OPIS_OPERACJI_COL] and (account["id"] in row[self.NUMER_KONTA_COL]):
+            row[self.PAYEE_COL] = 'Transfer: {}'.format(account["name"])
+
+    def rename_regularne_oszczedzanie(self, row, account):
+        if KONTO_REGULARNE["id"] in row[self.OPIS_OPERACJI_COL]:
+            row[self.PAYEE_COL] = 'Transfer: {}'.format(account["name"])
+
+    def replace_other(self, row, word):
+        if word in row[self.OPIS_OPERACJI_COL]:
+            row[self.PAYEE_COL] = word.capitalize()
+
+    def rename_credit_payup(self, row, card):
+        if SPLATA_KART in row[self.OPIS_OPERACJI_COL] and card["id"] in row[self.MEMO_COL]:
+            row[self.PAYEE_COL] = 'Transfer: {}'.format(card["name"])
+
+    def write_file_ynab(self, data, new_csv):
+        ynab_filename = YNAB_FILENAME_PREFIX + new_csv
+
+        with open(ynab_filename, 'w', encoding='utf-8') as converted_file:
+            csvWriter = csv.writer(converted_file, delimiter=YNAB_DELIMITER)
+            for row in data:
+                csvWriter.writerow(row)
+
+    def add_selected_cols_from_row_to_list(self, row, listing):
+        listing.append(
+            [
+                row[self.DATE_COL],
+                row[self.MEMO_COL],
+                row[self.PAYEE_COL],
+                row[self.AMOUNT_COL]
+            ]
+        )
+
+
+    def print_each_transaction_from(self, listing):
+        for transaction in listing:
+            print(transaction)
+        print("Parsing complete")
+
+class CreditCardParser(AccountParser):
```

Comment:
That looks like a code-smell, as `CreditCardParser` is most likely not a proper subclass of `AccountParser`, they just share some functions. You might want to extract those to `AbstractAccountParser` but if you were me, you'd just put them outside the class and forget about inheritance :-)